### PR TITLE
Update README.md for fedora

### PR DESCRIPTION
**Pull request by AmarOk1412**
I just make some changes in **README.md**
- multirust->rustup
- complete dependencies for fedora

If you want to test. I made a Dockerfile:

```
FROM fedora

RUN dnf update -y
# Install dependencies
RUN dnf install -y git libupnp-devel file tar sudo gcc \
openssl-devel avahi-devel sqlite-devel espeak-devel dbus-devel

# Install multirust
RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly
RUN git clone https://github.com/rust-lang-nursery/multirust-rs.git
WORKDIR multirust-rs
RUN cargo install
ENV PATH=/root/.cargo/bin:$PATH
RUN rustup-setup -y

# Clone repository
WORKDIR ..
RUN git clone https://github.com/fxbox/foxbox.git

# Build foxbox
WORKDIR foxbox
RUN rustup override add nightly-2016-03-07
RUN cargo build

# Run foxbox
EXPOSE 4000 4242
CMD ["cargo","run"]
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ rust:
 ```
 You should then be able to then use:
 ```
-multirust override nightly-2016-03-07
+rustup override add nightly-2016-03-07
```
Comment (fabricedesre) [inline README.md]: Why this change?

File: README.md [hunk 2]
```diff
@@ -39,10 +39,11 @@ to install and switch between versions of Rust.
...
 | `libupnp`    | `libupnp-dev`         | `libupnp-devel` | `libupnp`        | `libupnp`       |
-| `libssl`     | `libssl-dev`          | `?`             | via `base-devel` | `openssl`       |
-| `libavahi`   | `libavahi-client-dev` | `?`             | `avahi`          | `n.a.`          |
-| `libsqlite3` | `libsqlite3-dev`      | `?`             | `sqlite`         | `sqlite`        |
-| `libespeak`  | `libsespeak-dev`      | `?`             | `?`              | `?`             |
+| `libssl`     | `libssl-dev`          | `openssl-devel`             | via `base-devel` | `openssl`       |
+| `libavahi`   | `libavahi-client-dev` | `avahi-devel`             | `avahi`          | `n.a.`          |
+| `libsqlite3` | `libsqlite3-dev`      | `sqlite-devel`             | `sqlite`         | `sqlite`        |
+| `libespeak`  | `libsespeak-dev`      | `espeak-devel`             | `?`              | `?`             |
+| `libdbus`  | `?`      | `dbus-devel`             | `?`              | `?`             |
```
Comment (fabricedesre) [inline README.md]: Please keep the table aligned.

File: README.md (hunk 1 above)
Comment (AmarOk1412) [inline README.md]: Because I used rustup (https://github.com/rust-lang-nursery/multirust-rs#how-is-this-related-to-multirust)

Comment (fabricedesre) [inline README.md]: Well, let's wait until the "WARNING: This is an early beta. Expect breakage." disappears.

Comment (AmarOk1412) [inline README.md]: I agree.

Comment (fabricedesre) [inline README.md]: So can you remove this change and I'll merge? Thanks!

Comment (AmarOk1412) [inline README.md]: done.
